scripts/manage-build-cache.py

```
@@ -0,0 +1,361 @@
+        actions.append({
+            "kind": "cargo-package", "package": metrics["package"],
+            "estimated_bytes": metrics["bytes"],
+        })
+        if execute:
+            environment = os.environ.copy()
+            environment.update({
+                "MESH_LLM_CACHE_TARGET_DIR": str(target),
+                "MESH_LLM_CACHE_PACKAGE": metrics["package"],
+            })
+            result = subprocess.run(
+                ["just", "cache-cargo-clean"],
```

Ruff (0.16.1)

[error] Starting a process with a partial executable path

(S607)